pkgs/data/icons/beauty-line-icon-theme/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, gtk3, gnome-icon-theme, hicolor-icon-theme }:
+
+stdenv.mkDerivation rec {
+  pname   = "BeautyLine";
+  version = "0.0.1";
+
+  src = builtins.fetchTarball {
```

Comment:
Please use fetchzip.

Reply:
why? If I may ask

Reply:
It automatically extracts the content and it should correct permissions. Alternative fetchurl can be used but fetchTarball in an icon theme is very unusual.